Build/Build.cs

```diff
@@ -47,11 +48,23 @@ class Build : NukeBuild
     [PackageExecutable("nspec", "NSpecRunner.exe", Version = "3.1.0")]
     Tool NSpec3;
 
+#if OS_WINDOWS
+    const string NodeFramework = "win-x64";
+#elif OS_MAC
+    const string NodeFramework = "osx-x64";
+#else
+    const string NodeFramework = "linux-x64";
+#endif
+
+    [PackageExecutable("Node.js.redist", "node", Version = "16.17.1", Framework = NodeFramework)]
+    Tool Node;
```

Comment:
I failed to see that windows requires `node.exe`

Now I beginning to doubt a bit whether this becomes less readable than having the constants inlined
@dennisdoomen opinions?

```suggestion
#if OS_WINDOWS
    const string NodeFramework = "win-x64";
    const string NodeExecutable = "node.exe";
#elif OS_MAC
    const string NodeFramework = "osx-x64";
    const string NodeExecutable = "node";
#else
    const string NodeFramework = "linux-x64";
    const string NodeExecutable = "node";
#endif

    [PackageExecutable("Node.js.redist", NodeExecutable, Version = "16.17.1", Framework = NodeFramework)]
    Tool Node;
```

Reply:
Defining them separately actually makes it _less_ readable. Just in-line them please...

Reply:
Like my latest commit?